client/src/app/modules/module-instance/components/fields/date-field/date-field.component.ts

```diff
@@ -11,6 +14,11 @@ interface DateData extends FieldData {
   styleUrls: ['./date-field.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class DateFieldComponent extends FieldComponent<DateData> {
-  startDate = new Date(1995, 0, 1);
+export class DateFieldComponent extends FieldComponent<DateData>
+  implements OnInit {
+  startDate: any;
+
+  ngOnInit() {
+    this.startDate = this.cData.startAt || new Date();
```

Comment:
if `this.cData.startAt` exists we need to turn it into a date because it will come in as a number or string